draw.it.client/src/pages/host/HostScreen.jsx

```diff
@@ -164,8 +165,8 @@ function HostScreen() {
                                 <th>Name</th>
                             </tr>
                         </thead>
-                        <tbody>
-                            {joinedPlayers.map((player, index) => (
+                        <tbody>                            
+                        {joinedPlayers.map((player, index) => (
```

Comment:
```suggestion
                        <tbody>
                            {joinedPlayers.map((player, index) => (
```
What is this change for :D

Also, it breaks the DOM structure (because of the wrong tabulation)

Reply:
Yeah I dont know why github shows like this, but yeah I added tabulation so it would look better.